Sources/Types.swift

```diff
@@ -164,6 +164,13 @@ public enum FitType {
      sizeThatFits(..) method result.
      */
     case heightFlexible
+
+    /**
+     Adjust the view's size based on it's content size requirements so that it uses the
```

Comment:
Could you also update the Readme.md `sizeToFit()` documentation?

Reply:
Done